media-libs/netpbm/files/netpbm-10.86.21-misc-deps.patch

```diff
@@ -0,0 +1,70 @@
+Remake netpbm-10.76.00-misc-deps.patch
+
+do not autoprobe deps when we disabled them
+
+--- a/GNUmakefile
++++ b/GNUmakefile
+@@ -75,7 +75,10 @@
+ 
+ PROG_SUBDIRS = converter analyzer editor generator other
+ PRODUCT_SUBDIRS = lib $(PROG_SUBDIRS)
+-SUPPORT_SUBDIRS = urt icon buildtools test
++SUPPORT_SUBDIRS = icon buildtools test
++ifeq ($(URTLIB),$(BUNDLED_URTLIB))
++SUPPORT_SUBDIRS += urt
++endif
+ 
+ SUBDIRS = $(PRODUCT_SUBDIRS) $(SUPPORT_SUBDIRS)
+ 
+--- a/converter/other/Makefile
++++ b/converter/other/Makefile
+@@ -7,6 +7,7 @@
+ 
+ include $(BUILDDIR)/config.mk
+ 
++ifneq ($(XML2_LIBS),NONE)
```

Comment:
pretty sure we don't need this anymore.  we should be able to force `TEST_PKGCONFIG_LIBXML2=...` in the ebuild instead.

Reply:
This part of the Makefile decides whether to use *pkg-config* or *xml2-config* to find `libxml2`.  Without the patch it would autodetect it regardless of the USE flag.  Same later for png and X.